playbooks/byo/rhel_subscribe.yml

```diff
@@ -3,7 +3,7 @@
   tags:
   - always
 
-- include: ../common/openshift-cluster/std_include.yml
+- include: ../common/openshift-cluster/evaluate_groups.yml
```

Comment:
evaluate_groups.yml is not even needed here either.  This line can just be removed.  See https://github.com/openshift/openshift-ansible/pull/4045/files#diff-377065f5e66159003edb0fc50ed24ee4R11